storage/grpc_client.go

```diff
@@ -1872,6 +1872,7 @@ func (w *gRPCWriter) uploadBuffer(recvd int, start int64, doneReading bool) (*st
 
 	// Send a request with as many bytes as possible.
 	// Loop until all bytes are sent.
+sendBytes:
```

Comment:
This is cool but I've literally never seen this in Go code before. Can you add a clarifying comment? It's just pretty rare syntax.